members/CusOa638b05a1698944365.yaml

```diff
@@ -0,0 +1,117 @@
+bioguide: CusOa638b05a1698944365
+contact_form:
+  method: post
+  driver: chrome
+  steps:
+    - visit: "https://eplanning.blm.gov/eplanning-ui/project/57323/595/8004057/comment"
+    - find:
+        - selector: "#comment"
+    - fill_in:
+        - name: Message
+          selector: "#comment"
+          value: $MESSAGE
+          required: true
+        - name: First Name
+          selector: "#first-name"
+          value: $NAME_FIRST
+          required: false
+        - name: Last Name
+          selector: "#last-name"
+          value: $NAME_LAST
+          required: false
+    - find:
+        - selector: "#additional-info"
+    - click_on:
+        - selector: "#additional-info"
+    - find:
+        - selector: "#line1"
+    - fill_in:
+        - name: Street Address
+          selector: "#line1"
+          value: $ADDRESS_STREET
+          required: false
+        - name: City
+          selector: "#city"
+          value: $ADDRESS_CITY
+          required: false
+    - select:
```

Comment:
It doesn't look to me like the state selector is working - probably because this isn't a real html select